pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -168,6 +168,30 @@ func updateOrRemove(ctx *zedmanagerContext, status types.AppInstanceStatus) {
 	}
 }
 
+// diskSizeReservedForDom0
+// Return Values:
+//  deviceDiskSize, diskReservedForDom0, error
```

Comment:
listing some names of the return values doesn't tell me what they are i.e., their semantics.
Returns size of PersistDir in bytes and the number of bytes reserved for dom0.
The name is also confusing.

However, since all the caller cares about is one number: allowedDeviceDiskSizeForApps you can have a function with that name which returns a single number.

Reply:
Changed it as PersistDirTotalAndDom0ReservedSizes